rosbag2_cpp/include/rosbag2_cpp/player_clock.hpp

```diff
@@ -87,6 +87,20 @@ class PlayerClock
   ROSBAG2_CPP_PUBLIC
   double get_rate() const;
 
+  /**
+   * Set the paused state of the clock.
+   * If currently in a `sleep_until`, this will wake the sleep to return false,
+   * so that the caller is not hanging forever.
+   */
+  ROSBAG2_CPP_PUBLIC
+  void set_paused(bool paused);
```

Comment:
sorry for the premature comment here, but `set_paused` reads kinda funky to me. 
What about simply having `void pause()` and `void resume()`? That sounds a bit more natural to me.

Similar, `get_paused` could be renamed to `is_paused()`.

Reply:
I'm fine with that API - thanks for the early feedback